tt_metal/impl/dispatch/command_queue.hpp

```diff
@@ -453,11 +453,6 @@ class CommandQueue
     public:
         CommandQueue () = delete;
         CommandQueue ( Device * device, uint32_t id) : device_ptr(device), cq_id(id){}
-        CommandQueue(const CommandQueue&) = default;
-        CommandQueue(CommandQueue&&) = default;
-        CommandQueue& operator=(const CommandQueue&) = default;
-        CommandQueue& operator=(CommandQueue&&) = default;
-
```

Comment:
fyi: @TT-billteng

Reply:
(This was also from the LLVM 16 commit)

Reply:
This class has `std::mutex` inside it which makes the class move only and forces you to manually write a move constructor.  Just removing these default constructors makes the build happy.